code/game/objects/effects/landmarks_events.dm

```diff
@@ -82,9 +84,8 @@ Admin verb is called by code\modules\admin\verbs\event_triggers.dm
 
 /obj/effect/landmark/event_trigger/auto_narrate
 	var/message
-	var/isVis_or_isAud = 0	//0 for vis, 1 for aud
+	var/isPersonal_orVis_orAud = 0	//0 for personal, 1 for vis, 2 for aud
```

Comment:
The only thing I'll say is that this name is way too wordy. That, again, is subjective/personal opinion, but something far simpler, like narrate_mode would be easier to understand. But one var is still better.

Reply:
It's wordy, but the code it's used in is fairly short and it stays within like 68 colums or whatever the standard is (it looks fine in my code editor on desktop)

As it is not a var that'll ever really be reused outside of this feature - I think it should be fine to stay as is.

I did think of setting up some preprocessor #defines but I felt it may be too much for sth like this.